egs/wsj/s5/utils/mkgraph.sh

```diff
@@ -58,6 +56,14 @@ for f in $required; do
   [ ! -f $f ] && echo "mkgraph.sh: expected $f to exist" && exit 1;
 done
 
+N=`tree-info $tree | grep "context-width" | cut -d' ' -f2`
```

Comment:
You should check for exit status here e.g. by adding || exit 1, in case the path is bad or Kaldi is not compiled.
I prefer $() over `` for new code.